src/layouts/partials/head.html

```diff
@@ -22,6 +22,10 @@
   <meta content="summary" name="twitter:card">
   <meta content="width=device-width, initial-scale=1" name="viewport">
 
+  <link rel="preload" href="/fonts/SourceSansPro-Regular/SourceSansPro-Regular.ttf.woff2" as="font" type="font/woff2" crossorigin="anonymous">
```

Comment:
I'm not sure if these `preloads` will add any tangible benefit - the font files should be requested very early in the loading of the webpages